papis/commands/update.py

```diff
@@ -152,24 +153,83 @@
 logger = papis.logging.get_logger(__name__)
 
 
-def try_parsing_str(key: str, value: str) -> str:
+def _try_parsing_str(key: str, value: str) -> Any:
     """
     Tries to parse the input string as a python expression.
 
     :returns: The parsed input string if string is a python expression,
         otherwise an unchanged string.
     """
     try:
-        value = ast.literal_eval(value)
+        result = ast.literal_eval(value)
     except (SyntaxError, ValueError):
         logger.debug("Value '%s' of key '%s' is not a python expression.", value, key)
-    return value
+        result = value
+
+    return result
+
+
+def _move_files(directory: str, src: str, dst: str) -> str:
+    if src == dst:
+        return dst
+
+    src = os.path.join(directory, src)
+    if not os.path.exists(src):
+        return dst
+
+    from papis.paths import _make_unique_file
+    dst = _make_unique_file(os.path.join(directory, dst))
+
+    import shutil
+    shutil.copyfile(src, dst)
+
+    return os.path.basename(dst)
+
+
+def _parse_file_name(document: DocumentLike, value: str) -> tuple[int, str] | None:
```

Comment:
TODO: This needs to be a lot smarter about extensions, preferably using something from `papis.paths` 🤦